tests/data/experimental_combination/icone/wzdx_1.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Are these "wzdx_xxx" named files the planned events we want to combine with? Should we name them something other than wzdx_1/2/3 to be more specific about what data we're testing with?

Reply:
Renamed and updated tests